docs/rules/no-constructor-return.md

```diff
@@ -0,0 +1,50 @@
+# Disallow returning value in constructor (no-constructor-return)
+
+In JavaScript, returning a value in the constructor of a class may be a mistake. Forbidding this pattern prevents mistake resulting from unfamiliarity with the language or copy-paste error.
```

Comment:
```suggestion
In JavaScript, returning a value in the constructor of a class may be a mistake. Forbidding this pattern prevents mistakes resulting from unfamiliarity with the language or a copy-paste error.
```

Reply:
hmm, these should have been covered in my above resolved suggestion